### DISCUSSION TITLE
[Octernship] Need more information about the nature of UI library engineering task 1

### DISCUSSION THREAD

**Discussion by Bmg09** [community/community#51456] [GitHub Education] [answered]
Hey, I would love to work on this feature but before that I need to enquire about a couple of pointers.

- Do we currently have any designs for the UI library.
- Do we have to implement a 'login with' kind of feature for app write (talking about authentication bit here).
- Do we have an auth provider to link the library with.

What type of ui should we build do we need to use googlesigninclient or simply create a UI library for email and password authentication

**Comment by Ntshangase** [comment]
Hey,

For questions regarding Octobership tasks it recommended that you ask those question to your mentor,  or the people in charge of that  specific project.

**Reply by BenjaminJamesPiersonJr** [reply]
I'm trying to get all the historical old jewelry artwork history in England that going to reflect off to the United Kingdom and be held in a museum in Rome were full protection

**Comment by samus-aran** [comment, marked as answer]
Hi @Bmg09 👋 

We would recommend students working on Octernships projects to reach out to the project’s maintainers to ask for clarification on the proposed tasks or identify any issues they are experiencing. All project’s may have different preferred methods of communication but here are a few places to check where best to raise inquiry:

- Octernships Project’s README - This may include instructions on how and where to raise an inquiry.
- Octernships Project’s Discussions - Not all projects’ repos have this enabled. We encourage students to first search in case your question has already been answered.
- Octernships Project’s Issues - We encourage students to first search in case your question has already been answered. We also encourage students to follow instructions on the issue template(s) on how to create a suitable issue if provided.

Hope this helps 👍

**Reply by Gaellll** [reply]
.

**Reply by BenjaminJamesPiersonJr** [reply]
Be safe but aware because you might see something else that valuable around the old valuable currency all work are history of the Bible that's why it's kept together because all of it is very valuable

**Reply by Fitzjust4ullc** [reply]
Ok

**Reply by mothergroupltd1** [reply]
My account financial services online

**Comment by devyanikshirsagar** [comment]
where do i get the open source projects

**Reply by m5555a** [reply]
Thanks for all your accepted

**Reply by BenjaminJamesPiersonJr** [reply]
You welcome that's to much!

**Comment by Gaellll** [comment]
.

**Comment by rythmn1111** [comment]
Hey when I am seeing the projects for octernship it is showing "The application window has closed for all current projects. New projects will be added soon." does this means that octernships are over? or projects would be added soon? does anyone have idea when projects would be added?

**Reply by BenjaminJamesPiersonJr** [reply]
No just a new confirmed mind frame on what we are really looking for in this order ship to put order all around the world to build structures and guide lines to prevard destrition in mankind to with hold humanity and down shift and the kingdom to person finding positions in with people can work together and run a civilization with our our intervening and pay for our advice to start together our our way by law of heaven thank you too much!

**Comment by BenjaminJamesPiersonJr** [comment]
Hold your on money in a safe protected on you owned property first alright!

**Reply by yahyaalhass** [reply]
> Hold your on money in a safe protected on you owned property first alright!

**Comment by naysaiaung1** [comment]
[](url)

**Reply by yahyaalhass** [reply]
>

**Comment by naysaiaung1** [comment]
https://github.com/orgs/community/discussions/51456#discussioncomment-10849286

**Reply by yahyaalhass** [reply]
> https://github.com/orgs/community/discussions/51456#discussioncomment-10849286

**Comment by naysaiaung1** [comment]
![Uploading Screenshot_2024-10-21-10-18-02-891_com.ton_keeper.jpg…]()
